t/25-cache.t

```diff
@@ -51,7 +51,7 @@ use POSIX '_exit';
 use Mojo::IOLoop::ReadWriteProcess 'process';
 use Mojo::IOLoop::ReadWriteProcess::Session 'session';
 use OpenQA::Test::Utils qw(fake_asset_server wait_for_or_bail_out);
-use OpenQA::Test::TimeLimit '20';
+use OpenQA::Test::TimeLimit '60';
```

Comment:
Hm, so does it take really three times as long as in before? We should care to have speedy tests. Do you think you can maybe take some shortcuts in tests or mock to speed up?

Reply:
> so does it take really three times as long as in before?

I've already noticed that the runtime of some tests is too low when running them locally. I suppose it only works within the CI due to the scale factor. Maybe that's another one of these cases.

Reply:
No, i was just being generous because it took longer than 20 seconds in my VM to run the test before i even added anything new.

Reply:
But good to know i'm not the only one running into local tests being slower than on CI, even on a fast machine.

Reply:
> running into local tests being slower than on CI

As mentioned, the CI has the scale factor. So the limits are just higher within the CI; it is most likely not faster within the CI.

Reply:
Ah, i missed that there's a scale factor. I'll try reducing the value a bit then.

Reply:
Actually, set it to the local execution time plus a small margin. The local execution should not rely on any scaling factor (unless you want to gather coverage data locally).

Reply:
We even have the test time limits documented but the scaling aspect was not covered yet so I did that now in https://github.com/os-autoinst/openQA/pull/3499